server/routes/api/v1/postUsers.js

```diff
@@ -0,0 +1,137 @@
+const admin = require('firebase-admin');

```

Comment:
Same question here from previous file. Seems like it should be renamed to reflect that it works on singular users.

I also wonder if we should rename this file to something like "createUser.js" which is a little more obvious in my opinion than "getUser.js". I'll leave that up to you though.

Reply:
Yeah, I'll rename this file as well. 

One thing I'm not sure about is if naming the files differently than the routes they contain would lead to problems like not knowing what routes are supported without diving into each file. I suppose you could lean on API documentation to prevent duplicating a route in the future. 

Not sure the best approach, but for now I think I'll rename it to `createUser.js`.